Vagrantfile

```diff
@@ -160,15 +179,25 @@ Vagrant.configure(2) do |config|
     set -o errexit
     echo 'if test -e /mnt/hgfs ; then /mnt/hgfs/scf/bin/dev/setup_vmware_mounts.sh ; fi' >> .profile
 
-    echo 'export PATH=$PATH:/home/vagrant/scf/container-host-files/opt/scf/bin/' >> .profile
-    echo 'test -f /home/vagrant/scf/personal-setup && . /home/vagrant/scf/personal-setup' >> .profile
+    echo 'export PATH="${PATH}:#{HOME}/scf/container-host-files/opt/scf/bin/"' >> .profile
 
-    echo -e '\nexport HISTFILE=/home/vagrant/scf/output/.bash_history' >> .profile
+    echo -e '\nexport HISTFILE="#{HOME}/scf/output/.bash_history"' >> .profile
 
     # Check that the cluster is reasonable.
-    /home/vagrant/scf/bin/dev/kube-ready-state-check.sh
+    #{HOME}/scf/bin/dev/kube-ready-state-check.sh
+
+    direnv exec #{HOME}/scf make -C #{HOME}/scf copy-compile-cache
+  SHELL
+
+  # Provision the custom config scripts and personal setup.
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    set -o errexit
+
+    if [ -d "#{mounted_custom_config_scripts}/provision.d" ]; then
+      find "#{mounted_custom_config_scripts}/provision.d" -iname "*.sh" -exec "{}" \\;
```

Comment:
Does this need a `-executable`?
Being able to order this would be nice too (since some scripts may depend on other scripts)'